src/ontology/config/prefixes.csv

```diff
@@ -3,7 +3,7 @@ rdf,http://www.w3.org/1999/02/22-rdf-syntax-ns#
 rdfs,http://www.w3.org/2000/01/rdf-schema#
 xsd,http://www.w3.org/2001/XMLSchema#
 owl,http://www.w3.org/2002/07/owl#
-oio,http://www.geneontology.org/formats/oboInOwl#
```

Comment:
## `oio` removed from `prefixes.csv`

Reply:
> The negative is that now OAK is not checking for lexical matches on synonyms identified by the oio:hasExactSynonym field.

This makes no sense. 

I will look into it...

Reply:
@matentzn If I understand you correctly, that's what I'm saying. If `prefixes.csv` contains `oio`, then when I run the pipeline, it does matches on just on `rdfs:label` but also `oio:hasExactSynonym`, and you can see that reflected in my [subsample outputs](https://docs.google.com/spreadsheets/d/1topWKAf4-biD_dbmies-P3KQB53MKlovxIpUw0EYfDI/edit#gid=1579419867). And if I remove `oio` from `prefixes.csv` I no longer see those in the outputs.
So if we want to keep matching on `oio:hasExactSynonym`, it seems that we should keep `oio` in `prefixes.csv`, even though that causes it to be non-bijective. It's only passed to SemanticSQL anyway. Alternative is to ask OAK to rename `oio` to `oboInOwl`.

Reply:
_As of today here's where we are:_
Nico has looked into further and opened: https://github.com/INCATools/ontology-access-kit/issues/698

Joe looked to see other places in which `merged.db` is used for identifying possible side effects of changing `oio` to `oboInOwl` in `prefixes.csv` when constructing `merged.db`. These other usages were found, but do not know if unintended side effects could occur:
- `lexmatch-sssom-compare.py`
- `mappings.ipynb`
- `oak-lexmatch-sssom-compare.ipynb`
- `oak-lexmatch-sssom-compare-checkpoint.ipynb`
- `remove-mappings-test.ipynb`
- `synchronizeMappingsWithMondo-checkpoint.ipynb`

Nico said he would put in some time to solve this, so Joe is pausing activity on this.